wagtail/admin/tests/pages/test_explorer_view.py

```diff
@@ -581,6 +581,57 @@ def test_filter_by_date_updated(self):
         page_ids = {page.id for page in response.context["pages"]}
         self.assertEqual(page_ids, {self.new_page.id, new_page_child.id})
 
+    def test_filter_by_owner(self):
+        barry = self.create_user(
+            "barry", password="password", first_name="Barry", last_name="Manilow"
+        )
+        self.create_user(
+            "larry", password="password", first_name="Larry", last_name="King"
+        )
+
+        new_page_child = SimplePage(
+            title="New page child",
+            slug="new-page-child",
+            content="new page child",
+            owner=barry,
+        )
+        self.new_page.add_child(instance=new_page_child)
+
+        response = self.client.get(
+            reverse("wagtailadmin_explore", args=(self.root_page.id,)),
+        )
+        self.assertEqual(response.status_code, 200)
+        # Only users who own any pages should be listed in the filter
+        self.assertContains(response, "Barry Manilow")
+        self.assertNotContains(response, "Larry King")
+
+        response = self.client.get(
+            reverse("wagtailadmin_explore", args=(self.root_page.id,)),
+            {"owner": barry.pk},
+        )
+        self.assertEqual(response.status_code, 200)
+        page_ids = {page.id for page in response.context["pages"]}
+        self.assertEqual(page_ids, {new_page_child.id})
```

Comment:
Was wondering what would happen if you try to filter by a user that doesn't own any pages anyway (via the URL).

It turns out that the filters won't be applied and an error will be shown in the filter form, but it's somewhat hidden because it's in a dialog. I think this is something we should consider when building the popup component. Or at least later when we implement saved views. For now, I think it's OK to keep it out of this PR.